config.ts

```diff
@@ -1,5 +1,6 @@
 import * as path from 'path'
 import * as _ from 'lodash'
+import { argv } from 'yargs'
```

Comment:
this file gets used in a webpack bundling - and it is causing build issues. we probably can't use yargs in config.ts